config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -197,6 +197,51 @@ periodics:
       - --timeout=40m
       - --use-logexporter
 
+  # single node GKE storage experiments
+- name: ci-kubernetes-storage-scalability-storage-single-node-gke
```

Comment:
We're switching to the mode where gke tests are running on our internal prow.

Reply:
Should I close this and open the same PR on internal test infra?

Reply:
Yes please